packages/twenty-server/src/engine/core-modules/feature-flag/feature-flag.entity.ts

```diff
@@ -17,7 +17,7 @@ import { FeatureFlagKey } from 'src/engine/core-modules/feature-flag/enums/featu
 import { Workspace } from 'src/engine/core-modules/workspace/workspace.entity';
 
 @Entity({ name: 'featureFlag', schema: 'core' })
-@ObjectType('FeatureFlag')
+@ObjectType()
```

Comment:
@FelixMalfait @AMoreaux Not sure about this one `FeatureFlagEntity` name will be used instead of `FeatureFlag`.
I don't think we're directly using for now the entity type as a graphQL DTO, but it's better to keep the name for that one

Reply:
Or maybe we should just rename the class `FeatureFlagEntity` to `FeatureFlag` if we don't have any conflict with other types.

Reply:
I prefer to rename `FeatureFlagEntity` to `FeatureFlag` to avoid an exception for FeatureFlag.

WDYT @FelixMalfait?

Reply:
I'm ok with that, let's just double check if we have any conflicts

Reply:
Yes great let's rename `FeatureFlagEntity` to `FeatureFlag`